composer.json

```diff
@@ -48,7 +48,8 @@
         "typo3/cms-viewpage": "^9.5",
         "kitodo/presentation": "dev-master",
         "slub/slub-digitalcollections": "dev-master",
-        "slub/slub-web-sachsendigital": "dev-main"
+        "slub/slub-web-sachsendigital": "dev-main",
+        "typo3/cms-lowlevel": "^9.5"
```

Comment:
Just a really minor suggestion: Move the `typo3/cms-lowlevel` entry upwards to group it with the other `typo3` packages.